Doc/whatsnew/3.9.rst

```diff
@@ -759,6 +759,9 @@ Changes in the Python API
   environment variable when the :option:`-E` or :option:`-I` command line
   options are being used.
 
+* The constant values of future flags in the :mod:`future` module
+  is updated in order to prevent collision with compiler flags.
```

Comment:
Maybe explicitly mention that PyCF_ALLOW_TOP_LEVEL_AWAIT and CO_FUTURE_DIVISION constants had the same value?